kubernetes_deploy/staging/deployment.yml

```diff
@@ -60,14 +60,7 @@ spec:
             secretKeyRef:
               name: zendesk-api
               key: ZENDESK_API_USERNAME
-        - name: STATIC_FILES_BACKEND
-          value: s3
-        - name: AWS_STORAGE_BUCKET_NAME
-          valueFrom:
-            secretKeyRef:
-              name: s3
-              key: bucket_name
         - name: FEATURE_FLAG_NO_MAP
-          value: "true"
+          value: "false"
```

Comment:
I think if we merged this as `false`, anyone working on the new design of FALA will not see the new design page coming through on staging. So they'll turn it back on again in a different commit probably and merge it. 

I think this should be left `true`